src/bcf/mod.rs

```diff
@@ -37,6 +37,25 @@ pub struct Reader {
 unsafe impl Send for Reader {}
 
 
+/// Implementation for `Reader::set_threads()` and `Writer::set_threads`.
+pub fn set_threads(hts_file: *mut htslib::htsFile, n_threads: usize)
+        -> Result<(), ThreadingError> {
+    assert!(n_threads > 0, "n_threads must be > 0");
+
+    if n_threads == 0 {
```

Comment:
Please remove the if since this is already handled by the assertion above.

Reply:
Of course.